File: ts-sdk/src/sdk/models/operations/getanoauthtoken.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetAnOauthTokenPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=application_id" })
  applicationId: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=token_id" })
  tokenId: string;
}


export class GetAnOauthTokenRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetAnOauthTokenPathParams;
}


export class GetAnOauthTokenResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getAnOauthToken200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/getanoauthtoken.go
package operations

type GetAnOauthTokenPathParams struct {
	ApplicationID string `pathParam:"style=simple,explode=false,name=application_id"`
	Organization  string `pathParam:"style=simple,explode=false,name=organization"`
	TokenID       string `pathParam:"style=simple,explode=false,name=token_id"`
}

type GetAnOauthTokenRequest struct {
	PathParams GetAnOauthTokenPathParams
}

type GetAnOauthTokenResponse struct {
	ContentType                             string
	StatusCode                              int
	GetAnOauthToken200ApplicationJSONObject map[string]map[string]interface{}
}
